tests/mocha/block_test.js

```diff
@@ -433,7 +431,7 @@ suite('Blocks', function() {
             '</xml>'
         ), this.workspace);
         this.assertConnectionsEmpty();
-        this.clock.tick(1);
```

Comment:
Another one to file as a bug for future cleanup (and not handle in this PR): these calls are a good candidate for a helper function, which:
- takes in the text
- does the text to dom call
- appends dom to workspace
- asserts connections empty
- calls runAll

Reply:
Filed #4101